src/black/strings.py

```diff
@@ -217,6 +216,7 @@ def normalize_string_quotes(s: str) -> str:
             s = f"{prefix}{orig_quote}{body}{orig_quote}"
         new_body = sub_twice(escaped_orig_quote, rf"\1\2{orig_quote}", new_body)
         new_body = sub_twice(unescaped_new_quote, rf"\1\\{new_quote}", new_body)
+    # TODO: can probably be removed
```

Comment:
Probably only needed in pre-3.12 mode.

Reply:
removing it seems to fail tests, so i'm keeping the code.